### ISSUE TITLE
npm install doesn't work

### ISSUE THREAD

**Issue by aqeeburrahman** [flatlogic/light-blue-react-template#27]
npm install is not working through an error or unable to resolve dependencies ... 

`npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: light-blue-react-template@1.2.9
npm ERR! Found: eslint-plugin-flowtype@3.8.1
npm ERR! node_modules/eslint-plugin-flowtype
npm ERR!   dev eslint-plugin-flowtype@"3.8.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint-plugin-flowtype@"2.x" from eslint-config-react-app@4.0.1
npm ERR! node_modules/eslint-config-react-app
npm ERR!   dev eslint-config-react-app@"4.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
`

**Comment by Pegietix** [comment]
Second this issue!

**Comment by alokpurohit18** [comment]
Yeah neither npm nor yarn is working with the installation. same error for me as well.

**Comment by NunyaHotor** [comment]
I encountered the same problem .  Was any one of you been able been resolved  the problem ?

**Comment by patrick-steve** [comment]
@SoonYoung-Kwon I tried your solution. But still got an error.

`npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent`
